src/app/components/Header/index.js

```diff
@@ -6,13 +6,17 @@ import style from './Header.css'
 
 export default class Header extends Component {
   render() {
+    const { noMenu } = this.props
+
     return (
       <div className={style.header}>
         <NetworkSwitcher />
         <div className={style.titleBar}>
-          <div className={style.menuNavWrapper}>
-            <MainNav />
-          </div>
+          { noMenu ||
```

Comment:
Rather than running this check inline in the JSP, you could break it out and keep your template code a little cleaner.

`const menu = if (hasMenu) <div and mainMenu />`

Then in the JSP, something like `{ menu }` instead if the check. Keeping logic out of template code is best for testing when we get that running.

Reply:
Yeah, I like it.